velox/functions/prestosql/types/IPAddressType.cpp

```diff
@@ -17,11 +17,15 @@
 #include "velox/functions/prestosql/types/IPAddressType.h"
 #include <folly/IPAddress.h>
 #include "velox/expression/CastExpr.h"
+#include "velox/functions/prestosql/types/IPPrefixType.h"
 
 static constexpr int kIPV4AddressBytes = 4;
 static constexpr int kIPV4ToV6FFIndex = 10;
 static constexpr int kIPV4ToV6Index = 12;
 static constexpr int kIPAddressBytes = 16;
+static constexpr int kIPPrefixBytes = 17;
```

Comment:
I see multiple definitions in the cpp files for this and for kIPAddressBytes. Shouldn't the common ones be defined in the header file? Perhaps a common header that just defines these consts for IPADDRESS/IPPREFIX?

Reply:
Moved to IPAddressType.h since that should be the base class, as IPPREFIX contains an IPADDRESS.